python-package/fairgbm/sklearn.py

```diff
@@ -1033,3 +1199,51 @@ def fit(self, X, y,
                    "eval_at : iterable of int, optional (default=(1, 2, 3, 4, 5))\n"
                    f"{' ':12}The evaluation positions of the specified metric.\n"
                    f"{' ':8}{_early_stop}{_after_early_stop}")
+
+
+class FairGBMClassifier(LGBMClassifier):
+    """Helper wrapper to use FairGBM with an sklearn-like API."""
+
+    FAIRGBM_OBJECTIVE = "constrained_cross_entropy"
+
+    def __init__(self, boosting_type='gbdt', num_leaves=31, 
```

Comment:
Question, should we re-define the default values of LGBM key-word args here?

Perhaps it would be better to just list FairGBM-only args here and leave the rest to a `**kwargs` wildcard. That way all default values are maintained for common LGBM arguments.